src/main/java/com/mitchelltford/discordbot/Application.java

```diff
@@ -1,18 +1,13 @@
 package com.mitchelltford.discordbot;
 
+import java.io.IOException;
```

Comment:
```suggestion
import java.io.IOException;
```
Removed unused import